services/database/serverless.yml

```diff
@@ -1,19 +1,13 @@
 service: database
 
-frameworkVersion: "3"
+frameworkVersion: "4"
 
 plugins:
   - serverless-dynamodb
   - serverless-stack-termination-protection
-  - serverless-s3-bucket-helper
   - serverless-plugin-scripts
   - serverless-offline
 
-s3BucketHelper:
-  loggingConfiguration:
-    destinationBucketName: ${env:LOGGING_BUCKET, ssm:/configuration/${self:custom.stage}/s3/accessLogsBucket, ssm:/configuration/default/s3/accessLogsBucket}
-    logFilePrefix: ${env:LOGGING_BUCKET, ssm:/configuration/${self:custom.stage}/s3/accessLogsPrefix, ssm:/configuration/default/s3/accessLogsPrefix}
```

Comment:
We don't need these?

Reply:
I think this stuff was just copied in without understanding it. The serverless-s3-bucket-helper goes through the config for s3 buckets and enforces certain properties. In many cases I found, there was no s3 bucket in the serverless file so this was actually doing nothing ever.